symfony/security-bundle/6.4/config/routes/security.yaml

```
@@ -0,0 +1,3 @@
+logout:
```

```suggestion
_security_logout:
```

We always prefix things from Symfony with an underscore, to lower the chances of conflicting with userland code.